.travis/analyse.sh

```diff
@@ -9,4 +9,4 @@ trap finish EXIT
 
 make start wait-healthy
 
-make exec command="npx hydra-validator analyse http://localhost:8080/"
+make docker-compose command="exec -T app npx hydra-validator analyse http://localhost:8080/"
```

Comment:
Could make it an argument on Make's `exec`?

Reply:
No preference, will have an ugly name too (will copy from `docker-compose exec --help`)

Reply:
Prefer to isolate Docker Compose to the Makefile entirely.